app/tasks/maintenance.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+global $cli;
+
+require_once __DIR__.'/../init.php';
+
+use Appwrite\Database\Database;
+use Appwrite\Database\Document;
+use Appwrite\Database\Adapter\MySQL as MySQLAdapter;
+use Appwrite\Database\Adapter\Redis as RedisAdapter;
+use Appwrite\Database\Validator\Authorization;
+use Utopia\App;
+use Utopia\CLI\Console;
+use Utopia\Config\Config;
+
+
+function getConsoleDB() {
```

Comment:
I'll try and find a better way to manage these dependencies. Don't worry about it for now, but maybe add a //TODO comment

Reply:
Done